corehq/apps/app_manager/models.py

```diff
@@ -749,6 +749,7 @@ class FormBase(DocumentSchema):
     no_vellum = BooleanProperty(default=False)
     form_links = SchemaListProperty(FormLink)
     schedule_form_id = StringProperty()
+    case_references = ListProperty()
```

Comment:
Not ignoring this btw. I will implement a better solution (more like FormAction now) just taking a small break from case mgmt at the moment. Will put do not merge on it